src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
Just meant to say "value" instead of "object" is all. I like `false` as a cancellation value. For `undefined` I think you are suggesting it still fall back to `requestOptions` and my concern there is that it is not letting the user know that they likely made a mistake in their own code. Maybe I'm not understanding the use case, but my inclination would be to invoke the callback as having an error condition in the `undefined` case.